src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -286,11 +286,8 @@ public override async ValueTask<FormulaValue> Visit(CallNode node, EvalVisitorCo
                                 Kind = ex.ErrorKind
                             });
                     }
-
-                    if (IfFunction.CanCheckIfReturn(func))

```

Comment:
>if (IfFunction.CanCheckIfReturn(func)) [](http://example.com/codeflow?start=20&length=38)

PR #1179 is introducing Table function to also skip this Assert. No action needed. Just a comment to link one PR to the other.